### DISCUSSION TITLE
this is just a fork off oh my zsh

### DISCUSSION THREAD

**Discussion by paxcoder** [ohmybash/oh-my-bash#362] [General]
It is just made to work with oh my bash, probably all plugins are ported, IDK if creators agreed for that.

**Comment by akinomyoga** [comment]
> this is just a fork off oh my zsh
> It is just made to work with oh my bash,

It is already explicitly written in [README](https://github.com/ohmybash/oh-my-bash#license).

> probably all plugins are ported,

This is false. Many came from Oh My Zsh (OMZ), but not all the plugins are ported from OMZ. Many also came from Bash-it, some came from blogs writing about bash configurations, and some are made by the owner of OMB, @nntoan. Also, most importantly, we shouldn't neglect the contributions of the OMB users.
